### PR TITLE
Add ChangePassword Page

### PR DESCRIPTION
**Pull request by Bicik89**

### CODE REVIEW

File: public/Consts.js [hunk 1]
```diff
@@ -17,6 +17,9 @@ export const COLORS = {
   ANIMATION_START: '#DDD', // Subject to change
   ANIMATION_MIDWAY: '#EEE',
   ANIMATION_END: '#FFF',
+  FORM_FOCUS: '#325EA8',
+  FORM_ERROR: '#E8220C',
+  FORM_CORRECT: '#26A318',
```
Comment (242370) [inline public/Consts.js]: Change to NOTIFICATION_PRIMARY

File: src/components/organisms/FormChangePass.vue [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    }
+  },
+  validations() {
+    return {
+      oldPassword: { required, minLength: minLength(8), maxLength: maxLength(20) },
+      password: { required, minLength: minLength(8), maxLength: maxLength(20) },
+      passwordRepeat: { required, sameAsPassword: sameAs(this.password) },
+    }
+  },
+  // here also might want to move it
```
Comment (242370) [inline src/components/organisms/FormChangePass.vue]: Remove such comments

File: src/router.js [hunk 3]
```diff
@@ -26,6 +27,11 @@ const routes = [
     component: Offers,
     name: "Lista ofert - ShareIO"
   },
+  {
+    path: "/changepassword",
```
Comment (242370) [inline src/router.js]: Change to "/changePassword"

**Comment by Piotr-Skrobski** [comment]
1) In .input-row, remove the margin. It messes up with labels. Instead, add some class to all the inputs and add the margin-bottom to them.

2) Change color of input messages on error; they aren't visible

**Comment by Piotr-Skrobski** [comment]
Change watchers to this - solves issue with no message error if user hadn't inputted anything.

```
watch: {
  'v$.password.$model'() {
    if (!this.v$.password.required.$model || this.v$.password.minLength.$model || this.v$.password.maxLength.$model) {
      this.passwordError = "Hasło musi mieć od 8 do 20 znaków";
    } else {
      this.passwordError = '';
    }
  },
  'v$.passwordRepeat.$model'() {
    if (!this.v$.passwordRepeat.required.$model || this.v$.passwordRepeat.sameAsPassword.$model) {
      this.passwordRepeatError = "Hasła muszą się zgadzać";
    } else {
      this.passwordRepeatError = '';
    }
  },
},
```

**Comment by Piotr-Skrobski** [comment]
Will fix it tomorrow, just need to check JWT authentication before

**Review by Piotr-Skrobski — APPROVED** [review]

**Pull request merged**